### ISSUE TITLE
WhyNotWin11.com is not owned by Me

### ISSUE THREAD

**Issue by sanny-io** [rcmaehl/WhyNotWin11#66]
`https://www.whynotwin11.com`

https://i.imgur.com/6Rlz443.png

I haven't seen this mentioned anywhere. Is this from you @rcmaehl? The download link currently points to this repo's releases, but they might be trying to lay low for a while before switching it out 👀

**Comment by rcmaehl** [comment]
This website is not owned, operated, or affiliated with me. I'll see if I can get contact information via WHOIS or another means but definitely sketch. It's my fault for not buying the domain

**Comment by rcmaehl** [comment]
Hopefully this is someone trying to be helpful but I'll be reaching out to legal council and will make a case if this becomes an issue.

**Comment by krystian3w** [comment]
Maybe some hosts project ban page:

https://github.com/StevenBlack/hosts#list-of-all-hosts-file-variants

then may no load with NextDNS on default setup.

**Comment by krystian3w** [comment]
@yuki2718 can by marked as badware list this commercial domain? like these: https://github.com/uBlockOrigin/uAssets/issues/1738 https://github.com/uBlockOrigin/uAssets/issues/3060

@spirillen fyi

**Comment by rcmaehl** [comment]
The Reddit Account of the site owner has been found but has been inactive for a couple hours. The situation will be updated as it progresses.

**Comment by Yuki2718** [comment]
@krystian3w Need evidence the site is actually malicious or dangerous for us to block.

**Comment by rcmaehl** [comment]
@Yuki2718 I believe it's evening time for the current owner, however I've contacted them via several methods and are awaiting a reply.

**Comment by spirillen** [comment]
Thanks for your consideration @krystian3w 

However, I do  not currently see any threats from the domain, however I do believe there are missing a very very important disclaimer about the non-relation to @rcmaehl 

If we are scouting the site source code we will find that it yet another wp site and the biggest threat is google, facebook and nocookie.net

When all this said, yes it is ort that all traces of who owns this domain end in a closed reddit account

## My conclusion

If things on the site change, yes it would be added, as there currently are nothing dangerous on the site, i would leave it as it is.
It appears mostly like a friendly spider :space_invader: who helps  spreading the word. and the DL links should be monitored and the domain owner should should contact @rcmaehl 


<details><summary>HTML decoded</summary>

```html
<link rel='dns-prefetch' href='//fonts.googleapis.com' />
	<link rel='stylesheet' id='wp-block-library-css'  href='https://www.whynotwin11.com/wp-includes/css/dist/block-library/style.min.css?ver=5.7.2' type='text/css' media='all' />
<link rel='stylesheet' id='yasrcss-css'  href='https://www.whynotwin11.com/wp-content/plugins/yet-another-stars-rating/includes/css/yasr.css?ver=2.8.1' type='text/css' media='all' />
<link rel='stylesheet' id='google-fonts-style-css'  href='https://fonts.googleapis.com/css?family=Open+Sans%3A400%2C600%2C700%7CRoboto%3A400%2C500%2C700&#038;display=swap&#038;ver=10.3.8' type='text/css' media='all' />
</div></div></div><div id="pgc-gb14-60d86a683a89c-1-1"  class="panel-grid-cell" ><div id="panel-gb14-60d86a683a89c-1-1-0" class="widget_text so-panel widget widget_custom_html panel-first-child panel-last-child" data-index="2" ><div class="textwidget custom-html-widget"><blockquote class="twitter-tweet"><p lang="en" dir="ltr">WhyNotWin11 is a better replacement for Windows 11&#39;s PC Health Check &#8211; <a href="https://twitter.com/LawrenceAbrams?ref_src=twsrc%5Etfw">@LawrenceAbrams</a><a href="https://t.co/d5UbPNrzgV">https://t.co/d5UbPNrzgV</a></p>&mdash; BleepingComputer (@BleepinComputer) <a href="https://twitter.com/BleepinComputer/status/1408861525939531778?ref_src=twsrc%5Etfw">June 26, 2021</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script></div></div></div></div></div><script type="application/ld+json">{"@context":"https:\/\/schema.org\/","@type":"Product","name":"WhyNotWin11","description":"WhyNotWin11 Detection Script to help identify why your PC isn&#8217;t Windows 11 ready Download Version 2.2.2 &nbsp;\u2022&nbsp; 1&nbsp;MB [Total: 1 Average: 5] What is the WhyNotWin11 Tool? WhyNotWin11 is a tool that allows us to check whatever our Computer or Laptop can run Windows 11 or not. Unlike Microsoft&#8217;s Health Check Tool, This WhyNotWin11 Tool...","image":{"@type":"ImageObject","url":"","width":0,"height":0},"Review":null,"aggregateRating":{"@type":"AggregateRating","ratingValue":5,"ratingCount":1,"bestRating":5,"worstRating":1},"brand":"","sku":"","":""}</script>                        </div>
                                            </div>
.game-preloader-thumbnail {
    background-image: url(https://static.wikia.nocookie.net/fridaynightfunkin/images/7/70/FNF_Logo_ba-bumps.gif);
}
```

</details>

**Comment by pixeye33** [comment]
Since it's not mentioned by anyone yet...

The link to the .exe is no longer pointing to this repo's releases.

The .exe that the website offers does not have the same sha256 signature as the one found in releases.

https://www.virustotal.com/gui/ did not find anything.

**Comment by spirillen** [comment]
True...

```html
a class="maxbutton-1 maxbutton maxbutton-download" href="https://www.whynotwin11.com/WhyNotWin11.exe"><span class='mb-text'>Download</span></a></div>
	```

This makes it a malicious site., Thanks for the update @pixeye33

**Comment by rcmaehl** [comment]
This is a comment to provide verification of the @WhyNotWin11 twitter account

**Comment by pixeye33** [comment]
Correction : exe metadata says its 2.2.4 and the sha256 actually matches the one an the repo...

**Comment by micwoj92** [comment]
> Correction : exe metadata says its 2.2.4 and the sha256 actually matches the one an the repo...

Still, it makes no sense to rehost file (with wrong release number) It probably mirrors the link from readme
https://github.com/rcmaehl/WhyNotWin11/releases/latest/download/WhyNotWin11.exe

**Comment by rcmaehl** [comment]
@Yuki2718 situation has changed a bit. I can understand if you're still hesitant, and I'll tag you in any additional changes, but it appears they're specifically trying to hide the fact of the .com banner warning.

(Which is gonna be a bit hard to do if anyone checks for updates on the app.)

**Comment by rcmaehl** [comment]
Yep, the guy purged information about the owner and the placeholder pages where I had attempted to contact him. I'd rather not involve council butttttttttttt

**Comment by JourneyOver** [comment]
Going through the persons reddit posts (if I am looking at the correct person that is), it looks like they have done this type of stuff before with other things as well, so I wouldn't expect much from them as it looks like they are just up to no good and just taking advantage again of a popular application.

**Comment by rcmaehl** [comment]
> Going through the persons reddit posts (if I am looking at the correct person that is), it looks like they have done this type of stuff before with other things as well, so I wouldn't expect much from them as it looks like they are just up to no good and just taking advantage again of a popular application.

They have a few accounts ranging between 5-7 years old. All Generic Middle East names.

**Comment by JourneyOver** [comment]
> They have a few accounts ranging between 5-7 years old. All Generic Middle East names.

Ah the only one I've run across so far that has any mention of the website so far is an account named "chardasyaal", but makes sense that they would have multiple accounts. Hopefully you can get the site shut down :/

**Comment by colenh** [comment]
i contacted namecheap via a ticket to report abuse. hope this is solved soon!

**Comment by spirillen** [comment]
I'll be surprised if you can get the domain down.

But what I can't understand... Why are they putting up that domain, hosting an exe (Currently same sha256/md5).

What is there attempted gain here?? there are no banners/ads or severe tracking/spyware (Yet)

But what confuses me more is there usage of a complete WP setup for one page which could be written with have the bits for the frontpage, and not to mention the entire backend (db/wp-code, etc).

**Comment by rcmaehl** [comment]
> I'll be surprised if you can get the domain down.
> 
> But what I can't understand... Why are they putting up that domain, hosting an exe (Currently same sha256/md5).
> 
> What is there attempted gain here?? there are no banners/ads or severe tracking/spyware (Yet)
> 
> But what confuses me more is there usage of a complete WP setup for one page which could be written with have the bits for the frontpage, and not to mention the entire backend (db/wp-code, etc).

All I really needed from them was "Hey, I'm X. I'll be sure to keep it the latest build and note that I'm not affiliated somewhere on the page". I would have grabbed the .org anyway (and now have, DNS please propagate)

**Comment by spirillen** [comment]
Not the fasted response I ever seen... but voila

```
drill -T WhyNotWin11.org
.       518400  IN      NS      l.root-servers.net.
org.    172800  IN      NS      a2.org.afilias-nst.info.
whynotwin11.org.        86400   IN      NS      lennon.ns.cloudflare.com.
whynotwin11.org.        86400   IN      NS      martha.ns.cloudflare.com.
WhyNotWin11.org.        3600    IN      SOA     lennon.ns.cloudflare.com. dns.cloudflare.com. 2037648737 10000 2400 604800 3600
```

drill WhyNotWin11.org
;; ->>HEADER<<- opcode: QUERY, rcode: NOERROR, id: 43118
;; flags: qr rd ra ; QUERY: 1, ANSWER: 0, AUTHORITY: 1, ADDITIONAL: 0 
;; QUESTION SECTION:
;; WhyNotWin11.org.     IN      A

;; ANSWER SECTION:

;; AUTHORITY SECTION:
WhyNotWin11.org.        3600    IN      SOA     lennon.ns.cloudflare.com. dns.cloudflare.com. 2037648737 10000 2400 604800 3600

;; ADDITIONAL SECTION:

**Comment by JohnPlanetary** [comment]
So I'm guessing the banner will be updated to state that only whynotwin11.org is the official address, and will no longer list any new fake domain, since people can just keep creating new fake domains.

**Comment by RoadXY** [comment]
Virus total does give three vendors as malicious in the mean time, btw:
https://www.virustotal.com/gui/file/84b818bf3c06297fb59306bfde8a7afa2d3564483bac7b85ae8f0a4212bc7367/detection

**Comment by xmflsct** [comment]
https://www.whynotwin11.com/index.php/feed/ Hosted on cloudways?

**Comment by rcmaehl** [comment]
> So I'm guessing the banner will be updated to state that only whynotwin11.org is the official address, and will no longer list any new fake domain, since people can just keep creating new fake domains.

YEP

**Comment by iam-py-test** [comment]
> 
> 
> https://www.whynotwin11.com
> 
> https://i.imgur.com/6Rlz443.png
> 
> I haven't seen this mentioned anywhere. Is this from you @rcmaehl? The download link currently points to this repo's releases, but they might be trying to lay low for a while before switching it out 👀

Please edit your comment and put the domain in ` so it is not linked

**Comment by iam-py-test** [comment]
> 
> 
> @Yuki2718 can by marked as badware risk this commercial domain? like these: [uBlockOrigin/uAssets#1738](https://github.com/uBlockOrigin/uAssets/issues/1738) [uBlockOrigin/uAssets#3060](https://github.com/uBlockOrigin/uAssets/issues/3060)
> 
> @spirillen fyi

Reported to @DandelionSprout and can report elsewhere

**Comment by micwoj92** [comment]
> whynotwin

Thanks for suggestion, edited all comments with links.

**Comment by Yuki2718** [comment]
@iam-py-test We internally discussed. Unless the site directly endangers user by malware or scam, we don't block the site.

**Comment by iam-py-test** [comment]
> 
> 
> @iam-py-test We internally discussed. Unless the site directly endangers user by malware or scam, we don't block the site.

Does anyone know why Firefox and my Antivirus flagged the 2.2.2 version from GitHub but not the one from `https://www.whynotwin11.com/#download-buttons-os` even thought they are identical.

**Comment by krystian3w** [comment]
Rare downloads: https://blog.mozilla.org/security/2016/08/01/enhancing-download-protection-in-firefox/

**Comment by iam-py-test** [comment]
Why is the download from the WhyNotWin11[.]com website not flagged.

**Comment by krystian3w** [comment]
Because nobody reported to Google this page/urls as scam?

**Comment by Trinitek** [comment]
Noticed that `whynotwin11.org` does not redirect properly, but `www.whynotwin11.org` does.

**Comment by Kadantte** [comment]
> Noticed that `whynotwin11.org` does not redirect properly, but `www.whynotwin11.org` does.

I think it better if he uses `Cloudflare Page Rules` It's for free!

![image](https://user-images.githubusercontent.com/11579313/123732909-8b5ec200-d892-11eb-938c-4e8cfd5cf97f.png)

**Comment by rcmaehl** [comment]
> Noticed that `whynotwin11.org` does not redirect properly, but `www.whynotwin11.org` does.

Shows both working for me??? Unless I did something wrong. Tips are appreciated

![image](https://user-images.githubusercontent.com/716581/123733378-5f7b1680-d869-11eb-9eb1-7d331ed28a76.png)

**Comment by botflakes** [comment]
Redirecting fine on my end too

**Comment by Trinitek** [comment]
My version of Firefox wants to navigate to http://whynotwin11.org, but you only have a redirect for https. I would have to explicitly specify https _or_ use the www version.

**Comment by rcmaehl** [comment]
> My version of Firefox wants to navigate to http://whynotwin11.org, but you only have a redirect for https. I would have to explicitly specify https _or_ use the www version.

Any way to have http redirect to https so I can adjust the filters. Or a single rule that captures `http`, `https`, `www.`, and `whynotwin11.org`?

**Comment by Trinitek** [comment]
I don't know about the specific tool you're using, but I would try dropping the scheme so it's just `whynotwin11.org`, just like how the www entry doesn't have `http` or `https`. You may be able to specify a subdomain wildcard like `*.whynotwin11.org` so you only need one entry, but again that depends on your tool.

**Comment by Trinitek** [comment]
By the way... your `www` rule works for both `http` and `https`, so I do believe dropping the scheme off the non-`www` rule will do the trick.

**Comment by FadeMind** [comment]
Windows Defender Smart Screen (in Edge Chromium) blocked
![20210629-1624950821](https://user-images.githubusercontent.com/9846948/123753680-667e4500-d8ba-11eb-819a-4160c163b71a.png)

**Comment by iam-py-test** [comment]
> Defender SmartScreen (in Edge Chromium) blocked
> ![20210629-1624950821](https://user-images.githubusercontent.com/9846948/123753680-667e4500-d8ba-11eb-819a-4160c163b71a.png)

Tested www and non-www in Edge and no block. What domain did you test?

**Comment by FadeMind** [comment]
`https://www.whynotwin11.com`
![20210629-1624971995](https://user-images.githubusercontent.com/9846948/123802532-b4f80780-d8eb-11eb-802b-847e8896e212.png)

**Comment by iam-py-test** [comment]
> `https://www.whynotwin11.com`
> ![20210629-1624971995](https://user-images.githubusercontent.com/9846948/123802532-b4f80780-d8eb-11eb-802b-847e8896e212.png)

Ok. It must have been an issue on my end. Now I can reproduce the block

**Comment by spirillen** [comment]
> > Noticed that `whynotwin11.org` does not redirect properly, but `www.whynotwin11.org` does.
> 
> Shows both working for me??? Unless I did something wrong. Tips are appreciated
> 
> ![image](https://user-images.githubusercontent.com/716581/123733378-5f7b1680-d869-11eb-9eb1-7d331ed28a76.png)

First off, you need to use 308 (get = get), not 301(get become post) for redirect

```nginx
server {
    if ($host = mypdns.org) {
        return 308 https://$host$request_uri;
    } # managed by Certbot

        listen IP:80;
        server_name mypdns.org;
        return 308 https://$host$request_uri;
}

server {
    if ($host = mypdns.com) {
        return 308 https://$host$request_uri;
    } # managed by Certbot

         listen IP:80;
        access_log off;
        server_name mypdns.com;
        return 308 https://$host$request_uri;
}

server {
        listen IP:443 ssl http2;
        server_name www.mypdns.org www.mypdns.com mypdns.com;
        return 308 https://mypdns.org$request_uri;
        access_log off;
       ....
}

server {
        listen  IP:443 ssl http2;
        root /storage01/www/mypdns.org;
        server_name mypdns.org;
       ....
|
```

etc etc

**Comment by Kadantte** [comment]
> > My version of Firefox wants to navigate to http://whynotwin11.org, but you only have a redirect for https. I would have to explicitly specify https _or_ use the www version.
> 
> Any way to have http redirect to https so I can adjust the filters. Or a single rule that captures `http`, `https`, `www.`, and `whynotwin11.org`?

CD lets you force HTTPS too! Make your SSL/TLS a `FULL` then in Edge Certificates turn on `Always Use HTTPS` Cloudflare will redirect are the requests over http to https and you might want to turn on `Automatic HTTPS Rewrites` too. ppl think www is important but it's a subdomain for your website actually! You can create a CNAME that points to your domain root check the picture below!

![image](https://user-images.githubusercontent.com/11579313/123823750-618ab700-d8f5-11eb-9fa4-fa2cc71d0f0e.png)

Also my two link works just fine with or without www https://kadantte.moe/ - https://www.kadantte.moe/

**Comment by spirillen** [comment]
> CD lets you force HTTPS too! Make your SSL/TLS a FULL then

That's bad for performance as it is only redirecting. HTTP is cache-able HTTPs is not

HTTP redirect to one of www or !www for GH is the optimal for performance.

**Comment by Kadantte** [comment]
> > CD lets you force HTTPS too! Make your SSL/TLS a FULL then
> 
> That's bad for performance as it is only redirecting. HTTP is cache-able HTTPs is not
> 
> HTTP redirect to one of www or !www for GH is the optimal for performance.

This is just URL redirect so I don't see the performance anywhere here! just a domain working as Backlink nothing fancy to look for perfo also CD its perf is wat better than GH pages!

**Comment by iam-py-test** [comment]
You should add HTTPS or else someone can redirect it to somewhere else (I.e. a malware domain)
Security is more important than performance

**Comment by Kadantte** [comment]
> You should add HTTPS or else someone can redirect it to somewhere else (I.e. a malware domain)
> Security is more important than performance

This why I recommended Cloudflare for Free and more secure!

**Comment by spirillen** [comment]
> > > CD lets you force HTTPS too! Make your SSL/TLS a FULL then
> > 
> > 
> > That's bad for performance as it is only redirecting. HTTP is cache-able HTTPs is not
> > HTTP redirect to one of www or !www for GH is the optimal for performance.
> 
> This is just URL redirect so I don't see the performance anywhere here! just a domain working as Backlink nothing fancy to look for performance also CD its performance is way better than GH pages! But whatever...

It all in the header :smirk: 

```
curl -I 'http://whynotwin11.org'

HTTP/1.1 301 Moved Permanently
Date: Tue, 29 Jun 2021 17:08:57 GMT
Cache-Control: max-age=3600
Expires: Tue, 29 Jun 2021 18:08:57 GMT
Location: https://github.com/rcmaehl/WhyNotWin11/
cf-request-id: 0afa57e5c9000010ad4a279000000001
Report-To: {"endpoints":[{"url":"https:\/\/a.nel.cloudflare.com\/report\/v2?s=yXipVu%2B59itoOG3YfJOtNQlRQ4SZScWWhVJzt8XUr5dt0R9XrRXI07HSa6wv2Ca113AscnT6UMtIHddguMsoSLinMs1JdnehcFwxBk7Hah3ADFP6kodbCuwIYraY"}],"group":"cf-nel","max_age":604800}
NEL: {"report_to":"cf-nel","max_age":604800}
Server: cloudflare
CF-RAY: 6670c282dc0210ad-CPH
alt-svc: h3-27=":443"; ma=86400, h3-28=":443"; ma=86400, h3-29=":443"; ma=86400, h3=":443"; ma=86400
X-Cache: MISS from 
X-Cache-Lookup: MISS from firewall.matrix.lan:3128
Via: 1.1 firewall.matrix.lan (squid/4.15)
Connection: keep-aliv
```

This is a full header for a request true [SquidProxy](http://www.squid-cache.org/). Now lets brake it down to what is interesting do to the performance


```
HTTP/1.1 301 Moved Permanently
Cache-Control: max-age=3600
Expires: Tue, 29 Jun 2021 18:08:57 GMT

X-Cache: MISS from 
X-Cache-Lookup: MISS from firewall.matrix.lan:3128
```

On the second run you'll see it is now cached. (the same goes for most modern browsers actually)

```
X-Cache: HIT from firewall.matrix.lan
X-Cache-Lookup: HIT from firewall.matrix.lan:3128
```

Here is then the same, but over httpS, notice: there are no CACHE info, as secure protocols don't get cached

```
url -I 'https://whynotwin11.org'
HTTP/2 301 
date: Tue, 29 Jun 2021 17:12:47 GMT
cache-control: max-age=3600
expires: Tue, 29 Jun 2021 18:12:47 GMT
location: https://github.com/rcmaehl/WhyNotWin11/
cf-request-id: 0afa5b6818000010c50325a000000001
expect-ct: max-age=604800, report-uri="https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct"
report-to: {"endpoints":[{"url":"https:\/\/a.nel.cloudflare.com\/report\/v2?s=blAKQVmDsghJK7ejbX9OOUXkggtC%2BaYHo8PeJoqwB4pwDaljfpY8%2FrAofeNwS0QeMBsQU3We69vCDuZAB3%2F%2BZ16SuQZT%2FvTEv5Olok%2BLuERWbJirR8EQ%2FStesrMF"}],"group":"cf-nel","max_age":604800}
nel: {"report_to":"cf-nel","max_age":604800}
server: cloudflare
cf-ray: 6670c820297c10c5-CPH
alt-svc: h3-27=":443"; ma=86400, h3-28=":443"; ma=86400, h3-29=":443"; ma=86400, h3=":443"; ma=86400
```

You See the benefit now?

**Comment by iam-py-test** [comment]
> > > > CD lets you force HTTPS too! Make your SSL/TLS a FULL then
> > > 
> > > 
> > > That's bad for performance as it is only redirecting. HTTP is cache-able HTTPs is not
> > > HTTP redirect to one of www or !www for GH is the optimal for performance.
> > 
> > 
> > This is just URL redirect so I don't see the performance anywhere here! just a domain working as Backlink nothing fancy to look for performance also CD its performance is way better than GH pages! But whatever...
> 
> It all in the header 😏
> 
> ```
> curl -I 'http://whynotwin11.org'
> 
> HTTP/1.1 301 Moved Permanently
> Date: Tue, 29 Jun 2021 17:08:57 GMT
> Cache-Control: max-age=3600
> Expires: Tue, 29 Jun 2021 18:08:57 GMT
> Location: https://github.com/rcmaehl/WhyNotWin11/
> cf-request-id: 0afa57e5c9000010ad4a279000000001
> Report-To: {"endpoints":[{"url":"https:\/\/a.nel.cloudflare.com\/report\/v2?s=yXipVu%2B59itoOG3YfJOtNQlRQ4SZScWWhVJzt8XUr5dt0R9XrRXI07HSa6wv2Ca113AscnT6UMtIHddguMsoSLinMs1JdnehcFwxBk7Hah3ADFP6kodbCuwIYraY"}],"group":"cf-nel","max_age":604800}
> NEL: {"report_to":"cf-nel","max_age":604800}
> Server: cloudflare
> CF-RAY: 6670c282dc0210ad-CPH
> alt-svc: h3-27=":443"; ma=86400, h3-28=":443"; ma=86400, h3-29=":443"; ma=86400, h3=":443"; ma=86400
> X-Cache: MISS from 
> X-Cache-Lookup: MISS from firewall.matrix.lan:3128
> Via: 1.1 firewall.matrix.lan (squid/4.15)
> Connection: keep-aliv
> ```
> 
> This is a full header for a request true [SquidProxy](http://www.squid-cache.org/). Now lets brake it down to what is interesting do to the performance
> 
> ```
> HTTP/1.1 301 Moved Permanently
> Cache-Control: max-age=3600
> Expires: Tue, 29 Jun 2021 18:08:57 GMT
> 
> X-Cache: MISS from 
> X-Cache-Lookup: MISS from firewall.matrix.lan:3128
> ```
> 
> On the second run you'll see it is now cached. (the same goes for most modern browsers actually)
> 
> ```
> X-Cache: HIT from firewall.matrix.lan
> X-Cache-Lookup: HIT from firewall.matrix.lan:3128
> ```
> 
> Here is then the same, but over httpS, notice: there are no CACHE info, as secure protocols don't get cached
> 
> ```
> url -I 'https://whynotwin11.org'
> HTTP/2 301 
> date: Tue, 29 Jun 2021 17:12:47 GMT
> cache-control: max-age=3600
> expires: Tue, 29 Jun 2021 18:12:47 GMT
> location: https://github.com/rcmaehl/WhyNotWin11/
> cf-request-id: 0afa5b6818000010c50325a000000001
> expect-ct: max-age=604800, report-uri="https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct"
> report-to: {"endpoints":[{"url":"https:\/\/a.nel.cloudflare.com\/report\/v2?s=blAKQVmDsghJK7ejbX9OOUXkggtC%2BaYHo8PeJoqwB4pwDaljfpY8%2FrAofeNwS0QeMBsQU3We69vCDuZAB3%2F%2BZ16SuQZT%2FvTEv5Olok%2BLuERWbJirR8EQ%2FStesrMF"}],"group":"cf-nel","max_age":604800}
> nel: {"report_to":"cf-nel","max_age":604800}
> server: cloudflare
> cf-ray: 6670c820297c10c5-CPH
> alt-svc: h3-27=":443"; ma=86400, h3-28=":443"; ma=86400, h3-29=":443"; ma=86400, h3=":443"; ma=86400
> ```
> 
> You See the benefit now?


Why is better performance needed? The only point of this domain is to (correct me if I am wrong) redirect lost people to GitHub and prevent it from being bought up by malware authors. (Also may have the effect of redirecting people who made a typo away from the .com)
HTTPS prevents tampering, evil ISPs, and also increases user trust.

**Comment by spirillen** [comment]
@iam-py-test  wrote:

> Why is better performance needed? The only point of this domain is to (correct me if I am wrong) redirect lost people to GitHub and prevent it from being bought up by malware authors. (Also may have the effect of redirecting people who made a typo away from the .com)

I'm not sure I follow you here... we where talking about the optimal redirect setup for nginx (aka cloudflare) to this GH repo.

> HTTPS prevents tampering

Only IF the domain is
1. listed in googles HSTS list
2. The browser supports the HSTS lists
3. You can still overrule the HSTS by adding 0 to the HSTS header

For nginx
```nginx
add_header Strict-Transport-Security "max-age=0;" always;
```

> evil ISPs from spying, and also increases user trust.

?? they still see the DNS and http(s) request header... Only the content is hopefully encrypted...

**Comment by iam-py-test** [comment]
@spirillen

  1.  Why can the request override the HSTS preload? That seems like a major security issue
  2.  I agree HTTPS is probably not needed, unless
Someone uses whynotwin11.org as a way to get to the GitHub repo, so HSTS would have a benefit
Someday there is content hosted on this domain. (that could be done)
In my (non-expert) opinion, unless one of these happens, HTTPS redirects should not be used.
However, I think HTTPS should be setup (unless that is hard), as some browsers (i.e. Firefox) warn users against HTTP domains, and will try to upgrade everything to HTTPS.

Also, I think there should be a warning in the README.md (maybe at the top) that says all domains except whynotwin11.org which claim they are related to whynotwin11 are scams (like how uBlock Origin warns about the ublock.org scam in their README)
Might not help too much if users go to the wrong site in the first place, but might help a bit.

**Comment by spirillen** [comment]
> That seems like a major security issue

NO, not at all, the value lies in the controller of the server,  it's a way out for a miss configured cert, remember, humans makes mistakes.

> HTTP domains, and will try to upgrade everything to HTTPS

No for HTTP code 30(1|2|7|8) exactly for the cache HIT :smirk: 

> Someone uses whynotwin11.org as a way to get to the GitHub repo, so HSTS would have a benefit

Exactly how?? HSTS says only THIS Domain requires SSL connection.... but as it is redirecting, I don't see the need for HSTS at all, there are no pages, solely a HTTP redirect code!!

> However, I think HTTPS should be setup (unless that is hard)

Add this to the server section of the config...

```nginx
server {

        # SSL configuration
        ssl_stapling on;
        ssl_certificate /fullchain.pem;
        ssl_certificate_key privkey.pem;
        ssl_trusted_certificate chain.pem;
        resolver <IP> valid=600s;
        resolver_timeout 5s;

        add_header Strict-Transport-Security "max-age=31536000; preload" always;

}
```

Or simply  [Verifying or approving a domain for your organization](https://docs.github.com/en/organizations/managing-organization-settings/verifying-or-approving-a-domain-for-your-organization#verifying-a-domain-for-your-organization) it's kinda a single TXT record in the DNS zone.....

**Comment by thepwrtank18** [comment]
The build from whynotwin11.com is now modified to have their own website instead of whynotwin11.org, and the logo changed to an "A" of some sort.

Updates are still via the GitHub repo.

**Comment by rcmaehl** [comment]
![image](https://user-images.githubusercontent.com/716581/124015462-b3dddd80-d9b2-11eb-8650-4e934fdc7375.png)

I was gonna check but Avast had other plans

**Comment by micwoj92** [comment]
> The build from whynotwin11.com is now modified to have their own website instead of whynotwin11.org, and the logo changed to an "A" of some sort.
> 
> Updates are still via the GitHub repo.

I suspect that "A" is just default AutoIt icon, that's EPIC FAIL :sunglasses:

**Comment by rcmaehl** [comment]
I'll decompile it and take a look at their changes (if any)

**Comment by iam-py-test** [comment]
@rcmaehl You should be able to override, or you can maybe use something like `curl`

**Comment by spirillen** [comment]
> I'll decompile it and take a look at their changes (if any)

Just for other to see the sha512sum, as I don't see any reason for other to download a badware.exe on there Windos (I'm running *nix :smirk: )

```shell
sha512sum WhyNotWin11.exe WhyNotWin11.com.exe 
9cc044f91b9f2d7f818ecdf208ea2cac371c26de7724b0623b28c045eef2c2a6fc7025d096075d6d4cb95d53ec3dbf6d3410185adfa887c19658ff03ee3d0a72  WhyNotWin11.exe
f92d71e1ab3c61c1c1f9f02c70e564816dfa2a853af68a6ab834f156d90d71f45c94673da2eb1bbd29a31b7d671ccc1d46d1e63030226790f380bb6d92fc732b  WhyNotWin11.com.exe
```

**Comment by iam-py-test** [comment]
> > I'll decompile it and take a look at their changes (if any)
> 
> Just for other to see the sha512sum, as I don't see any reason for other to download a badware.exe on there Windos (I'm running *nix 😏 )
> 
> ```shell
> sha512sum WhyNotWin11.exe WhyNotWin11.com.exe 
> 9cc044f91b9f2d7f818ecdf208ea2cac371c26de7724b0623b28c045eef2c2a6fc7025d096075d6d4cb95d53ec3dbf6d3410185adfa887c19658ff03ee3d0a72  WhyNotWin11.exe
> f92d71e1ab3c61c1c1f9f02c70e564816dfa2a853af68a6ab834f156d90d71f45c94673da2eb1bbd29a31b7d671ccc1d46d1e63030226790f380bb6d92fc732b  WhyNotWin11.com.exe
> ```

You always can download in a sandboxed VM or on a Linux machine (my favorite VM-less way to examine malware)

**Comment by JeremyWu0212** [comment]
@rcmaehl That's the webpage.
![IMG_20210701_101359](https://user-images.githubusercontent.com/79089455/124054926-94fa3e00-d9e8-11eb-984d-b179e233a344.jpg)
疑似高仿网站？？？

**Comment by JeremyWu0212** [comment]
@rcmaehl The exe file's size is 686kb.
The file is download from www.whynotwin11.com

**Comment by rcmaehl** [comment]
686kb? I would not trust it at all then

**Comment by JeremyWu0212** [comment]
@rcmaehl And it maybe created by Babbu.
![IMG_20210701_102906](https://user-images.githubusercontent.com/79089455/124056098-b78d5680-d9ea-11eb-9c9d-4ef577e750b6.jpg)

**Comment by JeremyWu0212** [comment]
![IMG_20210701_104148](https://user-images.githubusercontent.com/79089455/124057006-58304600-d9ec-11eb-9a63-5653e1138864.jpg)
![IMG_20210701_104234](https://user-images.githubusercontent.com/79089455/124057007-58304600-d9ec-11eb-9723-38b9cd064c16.jpg)
@rcmaehl

**Comment by rcmaehl** [comment]
![image](https://user-images.githubusercontent.com/716581/124061954-d77a4580-d9fd-11eb-9aad-3a3c79225d07.png)

I can confirm that application doesn't seem to be an autoit file or at least no one that the trusted decompiler can recognize. Oddly enough, VirusTotal Behavior report shows it running some of the commands that should be expected. App was UPX Compressed which the decompiler can handle but it still didn't recognize it.

I wouldn't trust their as far as I can throw it.

**Comment by thepwrtank18** [comment]
You should probably take action by sending a DMCA complaint to Google (and other search engines) for impersonation and spreading malware.

**Comment by spirillen** [comment]
Google isn't the one who handle misplaced domain that IANA/RIPE

In this case the first contact however would be name:sheep: 

```
whois whynotwin11.com
   Domain Name: WHYNOTWIN11.COM
   Registry Domain ID: 2622571933_DOMAIN_COM-VRSN
   Registrar WHOIS Server: whois.namecheap.com
   Registrar URL: http://www.namecheap.com
   Updated Date: 2021-06-27T08:37:11Z
   Creation Date: 2021-06-27T08:22:51Z
   Registry Expiry Date: 2022-06-27T08:22:51Z
   Registrar: NameCheap, Inc.
   Registrar IANA ID: 1068
   Registrar Abuse Contact Email: abuse@namecheap.com
   Registrar Abuse Contact Phone: +1.6613102107
```

Then if they are not taking desired actions, then it RIPE: <https://duckduckgo.com/?t=ffab&q=ripe+report+domain&ia=web>

**Comment by Yuki2718** [comment]
@rcmaehl Okay, 27/70 in VT would be enough to judge the software is suspicious.

**Comment by CKylinMC** [comment]
I don't know should I create a new issue but I can't visit https://www.whynotwin11.org ...

![image](https://user-images.githubusercontent.com/11648014/124121503-89ba1780-daa7-11eb-9373-e0eeb70f9116.png)

**Comment by micwoj92** [comment]
Try without www, it's just a redirect to this repo.

**Comment by CKylinMC** [comment]
> Try without www, it's just a redirect to this repo.

OK, it works. Thanks!

**Comment by colenh** [comment]
keeps getting worse doesn't it, ive reported it to both cloudflare and namecheap but i havent recieved any replies.

**Comment by iam-py-test** [comment]
Report to uBlock Origin at https://github.com/uBlockOrigin/uAssets/issues
That will increase the blocking

**Comment by micwoj92** [comment]
Apparently namecheap is bad when it comes to replying to abuse report emails. No idea about cloudflare.

**Comment by iam-py-test** [comment]
> Apparently namecheap is bad when it comes to replying to abuse report emails. No idea about cloudflare.

If we report to mainstream blocklists, they will be more likely to respond

**Comment by micwoj92** [comment]
> Report to uBlock Origin at https://github.com/uBlockOrigin/uAssets/issues
> That will increase the blocking

https://github.com/uBlockOrigin/uAssets/commit/d2b2afe18f9cf3196b58b7a0e9fb67d1d83e5444

**Comment by iam-py-test** [comment]
Oh. Thanks!

**Comment by blocklistproject** [comment]
At this time we have a similar response to @spirillen, the site still currently links to the repo, after scanning the site we are not able to find any obvious malicious serving content and VirusTotal gives it a clean bill of health. Please keep us updated and as soon as this changes we will add it to our lists.

**Comment by iam-py-test** [comment]
@blocklistproject the download has a large number of detections on VirusTotal:

![](https://user-images.githubusercontent.com/716581/124061954-d77a4580-d9fd-11eb-9aad-3a3c79225d07.png)

**Comment by iam-py-test** [comment]
https://www.virustotal.com/gui/file/831c7d8b75738072e74020f61670ebde2e88b4f72cac941cdddf09d5de263a18/detection

**Comment by blocklistproject** [comment]
I am unable to download the file in a safe environment right now to confirm. So, out of an abundance of caution based on the image above we will at least be temporarily blocking the site. Once I am able to confirm this, the block will either be removed or will permanently be left in place. Thank you for bringing this to our attention and the quick reply regarding our concern. It takes a community!

**Comment by iam-py-test** [comment]
> I am unable to download the file in a safe environment right now to confirm. So, out of an abundance of caution based on the image above we will at least be temporarily blocking the site. Once I am able to confirm this, the block will either be removed or will permanently be left in place. Thank you for bringing this to our attention and the quick reply regarding our concern. It takes a community!

I posted a scan above

**Comment by micwoj92** [comment]
@blocklistproject At first that website was serving .exe from this repo, then they switched to self hosting and next day they had changed executable to show .com website instead of real whynotwin.org redirect. Then they changed to totally different exe which scan you can see above. It seems that now they changed to linking to official download again but it is uncertain for how long and what next action will owner of this fake website think. I think it should be blocked permanently. Thanks for considering.

**Comment by spirillen** [comment]
> At this time we have a similar response to @spirillen, the site still currently links to the repo ...

I'm a bit confused about the intention with this message? What is it you are trying to tell me? 
Could you please elaborate @blocklistproject ?

**Comment by dxgldotorg** [comment]
https://who.is/whois/whynotwin11.com

Bad news, the web host is hidden behind Cloudflare.

Perhaps you might be able to get in touch with NameCheap support (the registrar) but they might require a UDRP.

**Comment by iam-py-test** [comment]
> https://who.is/whois/whynotwin11.com
> 
> Bad news, the web host is hidden behind Cloudflare.
> 
> Perhaps you might be able to get in touch with NameCheap support (the registrar) but they might require a UDRP.

If enough places block it, it might not matter

**Comment by colenh** [comment]
> > https://who.is/whois/whynotwin11.com
> > Bad news, the web host is hidden behind Cloudflare.
> > Perhaps you might be able to get in touch with NameCheap support (the registrar) but they might require a UDRP.
> 
> If enough places block it, it might not matter
> Does anyone know how to get NameCheap to take it down?

already contacted them and cloudflare and im awaiting a response

**Comment by MDMCK10** [comment]
![image](https://user-images.githubusercontent.com/21245760/124283776-92185e00-db4c-11eb-88e3-415bf9751d7e.png)

**Comment by iam-py-test** [comment]
@MDMCK10 is this mean it has been taken down or will be taken down?
https://sitecheck.sucuri.net/results/whynotwin11.com shows it is still online (for now)

If it gets completely taken down, @rcmaehl should try to buy the domain before someone else uses it

**Comment by MDMCK10** [comment]
> 
> 
> @MDMCK10 is this mean it has been taken down or will be taken down?
> https://sitecheck.sucuri.net/results/whynotwin11.com shows it is still online (for now)
> 
> If it gets completely taken down, @rcmaehl should try to buy the domain before someone else uses it

This just means that Cloudflare has put a warning on the WhyNotWin11.exe file that's on the site, (and forwarded a complaint to the hosting provider as well) the website itself is still online, but I'm also working with Namecheap at the moment to get the domain itself taken down. I'll post updates as they happen.

**Comment by iam-py-test** [comment]
> 
> 
> > @MDMCK10 is this mean it has been taken down or will be taken down?
> > https://sitecheck.sucuri.net/results/whynotwin11.com shows it is still online (for now)
> > If it gets completely taken down, @rcmaehl should try to buy the domain before someone else uses it
> 
> This just means that Cloudflare has put a warning on the WhyNotWin11.exe file that's on the site, (and forwarded a complaint to the hosting provider as well) the website itself is still online, but I'm also working with Namecheap at the moment to get the domain itself taken down. I'll post updates as they happen.

@MDMCK10 Thank you!

**Comment by MDMCK10** [comment]
> Also, why is it look like gmail is warning you about that email?

the gmail warning works in strange ways sometimes - i forgot to click "looks safe" before screenshotting it

**Comment by MDMCK10** [comment]
Got response from Namecheap: "Thank you, we'll check it."

**Comment by MDMCK10** [comment]
Domain Status: clientHold https://icann.org/epp#clientHold

We win!

**Comment by micwoj92** [comment]
We did it Reddit!

**Comment by iam-py-test** [comment]
> 
> 
> We did it Reddit!

Reddit?

**Comment by micwoj92** [comment]
reddit.com

**Comment by iam-py-test** [comment]
> 
> 
> https://old.reddit.com

I know what Reddit is. 
I was just curious why you said Reddit when this is GitHub

**Comment by micwoj92** [comment]
Because @rcmaehl is notorious Redditor.

**Comment by MDMCK10** [comment]
> 
> 
> Domain Status: clientHold https://icann.org/epp#clientHold
> 
> We win!

... or so I thought
looks like the domain is (sadly) back up, except the "WhyNotWin11.exe" file on it is gone.
I guess it's up to @rcmaehl to take legal action at this point, since whoever is behind the website clearly wants to keep it up for some (likely malicious) reason

**Comment by iam-py-test** [comment]
> 
> 
> > Domain Status: clientHold https://icann.org/epp#clientHold
> > We win!
> 
> ... or so I thought
> looks like the domain is (sadly) back up, except the "WhyNotWin11.exe" file on it is gone.
> I guess it's up to @rcmaehl to take legal action at this point, since whoever is behind the website clearly wants to keep it up for some (likely malicious) reason

How did they do that?
Also, it is blocked by a whole bunch of places, so hopefully that will provide some protection.
Has anyone been able to get through to Google abuse? They seem to ignore my Safe Browsing reports

**Comment by MDMCK10** [comment]
> 
> 
> > > Domain Status: clientHold https://icann.org/epp#clientHold
> > > We win!
> > 
> > 
> > ... or so I thought
> > looks like the domain is (sadly) back up, except the "WhyNotWin11.exe" file on it is gone.
> > I guess it's up to @rcmaehl to take legal action at this point, since whoever is behind the website clearly wants to keep it up for some (likely malicious) reason
> 
> How did they do that?
> Also, it is blocked by a whole bunch of places, so hopefully that will provide some protection.
> Has anyone been able to get through to Google abuse? They seem to ignore my Safe Browsing reports

How did they do that? Well, registrars actually give you a way to get your domain back up if you resolve the "problem", which in this case was the "WhyNotWin11.exe" file on the site being considered malware, since the person got rid of that file, the registrar allowed the domain to be put back up.
I submitted to safe browsing, but nothing has happened.
As for how many vendors are blocking it, the answer is "some", but it's not enough, we would need Google SafeBrowisng blocking it for any major impact to happen.

**Comment by iam-py-test** [comment]
> 
> 
> > > > Domain Status: clientHold https://icann.org/epp#clientHold
> > > > We win!
> > > 
> > > 
> > > ... or so I thought
> > > looks like the domain is (sadly) back up, except the "WhyNotWin11.exe" file on it is gone.
> > > I guess it's up to @rcmaehl to take legal action at this point, since whoever is behind the website clearly wants to keep it up for some (likely malicious) reason
> > 
> > 
> > How did they do that?
> > Also, it is blocked by a whole bunch of places, so hopefully that will provide some protection.
> > Has anyone been able to get through to Google abuse? They seem to ignore my Safe Browsing reports
> 
> How did they do that? Well, registrars actually give you a way to get your domain back up if you resolve the "problem", which in this case was the "WhyNotWin11.exe" file on the site being considered malware, since the person got rid of that file, the registrar allowed the domain to be put back up.
> I submitted to safe browsing, but nothing has happened.
> As for how many vendors are blocking it, the answer is "some", but it's not enough, we would need Google Safe Browsing blocking it for any major impact to happen.

I am aware, but GSB's report system is horrible, and domains I reported months ago still are not blocked.
I am not even sure if I have the right url.
However, if we submit it to GSB enough times, and get other places onboard, we may be able to make an impact

**Comment by micwoj92** [comment]
> > Domain Status: clientHold https://icann.org/epp#clientHold
> > We win!
> 
> ... or so I thought
> looks like the domain is (sadly) back up, except the "WhyNotWin11.exe" file on it is gone.
> I guess it's up to @rcmaehl to take legal action at this point, since whoever is behind the website clearly wants to keep it up for some (likely malicious) reason

Is it? I cant connect to it.

**Comment by iam-py-test** [comment]
> 
> 
> > > Domain Status: clientHold https://icann.org/epp#clientHold
> > > We win!
> > 
> > 
> > ... or so I thought
> > looks like the domain is (sadly) back up, except the "WhyNotWin11.exe" file on it is gone.
> > I guess it's up to @rcmaehl to take legal action at this point, since whoever is behind the website clearly wants to keep it up for some (likely malicious) reason
> 
> Is it? I cant connect to it.

Just checked. I can't connect either

**Comment by rcmaehl** [comment]
![Screenshot_20210702-155454](https://user-images.githubusercontent.com/716581/124322671-dd7f3c00-db4d-11eb-820a-07a3245d5802.png)

I'll check some of my several tickets with several vendors and see if there's any update

**Comment by krystian3w** [comment]
On my ISP also is offline.

**Comment by iam-py-test** [comment]
@MDMCK10 maybe your seeing a cached version

**Comment by mlipok** [comment]
> On my ISP also is offline.
Chceck again.

It could be related to the recent changes:
https://github.com/rcmaehl/WhyNotWin11/issues/345

**Comment by MDMCK10** [comment]
> 
> 
> @MDMCK10 maybe your seeing a cached version

... oh the darn caches
i checked what happened, it looks like that it came back up for a brief period of time on my end for some reason, and when i was doing the WHOIS lookup it did not show clientHold
after making sure that nothing is being cached on my end, the site once again looks to be offline
(sorry about that, i did not notice at first)

**Comment by iam-py-test** [comment]
> 
> 
> > @MDMCK10 maybe your seeing a cached version
> 
> ... oh the darn caches
> i checked what happened, it looks like that it came back up for a brief period of time on my end for some reason, and when i was doing the WHOIS lookup it did not show clientHold
> after making sure that nothing is being cached on my end, the site once again looks to be offline
> (sorry about that, i did not notice at first)

Ok. Thanks for checking
We all make errors

**Comment by MDMCK10** [comment]
> 
> 
> > > @MDMCK10 maybe your seeing a cached version
> > 
> > 
> > ... oh the darn caches
> > i checked what happened, it looks like that it came back up for a brief period of time on my end for some reason, and when i was doing the WHOIS lookup it did not show clientHold
> > after making sure that nothing is being cached on my end, the site once again looks to be offline
> > (sorry about that, i did not notice at first)
> 
> Ok. Thanks for checking
> We all make errors

i blame my ISP for having bad DNS servers that keep stuff for way longer than they're supposed to
i thought i had it set up so i use cloudflare as DNS, but i guess i have to double check that now

**Comment by thepwrtank18** [comment]
It's down.
ICANN Lookup shows [clientHold](https://www.icann.org/resources/pages/epp-status-codes-2014-06-16-en#clientHold) and [clientTransferProhibited](https://www.icann.org/resources/pages/epp-status-codes-2014-06-16-en#clientTransferProhibited).

We won.

**Comment by spirillen** [comment]
> No registered domain for `whynotwin11.com` and for others how to test the trail of a domain
> 
> On Linux we have several of perfect tools for this.
> 
> 
> Drill is superior to trail DNS hierarchic
> 
> ```
> drill whynotwin11.com -T
> .       518400  IN      NS      e.root-servers.net.
> -- snip --
> com.    172800  IN      NS      m.gtld-servers.net.
> com.    900     IN      SOA     a.gtld-servers.net. nstld.verisign-grs.com. 1625296846 1800 900 604800 86400
> ```
> 
> Next you can use the WHOIS tool from jwhois
> 
> ```
> whois whynotwin11.com
>    Domain Name: WHYNOTWIN11.COM
>    Registry Domain ID: 2622571933_DOMAIN_COM-VRSN
>    Registrar WHOIS Server: whois.namecheap.com
>    Registrar URL: http://www.namecheap.com
>    Updated Date: 2021-07-02T14:11:59Z
>    Creation Date: 2021-06-27T08:22:51Z
>    Registry Expiry Date: 2022-06-27T08:22:51Z
>    Registrar: NameCheap, Inc.
>    Registrar IANA ID: 1068
>    Registrar Abuse Contact Email: abuse@namecheap.com
>    Registrar Abuse Contact Phone: +1.6613102107
>    Domain Status: clientHold https://icann.org/epp#clientHold
>    Domain Status: clientTransferProhibited https://icann.org/epp#clientTransferProhibited
>    Name Server: JERRY.NS.CLOUDFLARE.COM
>    Name Server: LILA.NS.CLOUDFLARE.COM
> 
> -- snip --
> 
> >>> Last update of WHOIS database: 2021-07-03T01:51:15.63Z <<<
> ```
> 
> Here you will see RIPE database of registered domains and to which DNS servers the record is bound to.
> 
> As a response to @thepwrtank18 "We won"...
> 
> We??  Won??
> Who are We?
> Were is the Win?
> 
> There is no winning in taking down a domain, it would have been a WIN if the people behind this repo have got in contact whit the people behind `whynotwin11.com` and a constructive dialog have come for the day.
> 
> Nobody has won anything, and the domain status is "only" put on hold until a dispute is RE-solved. In short English, The domain is on pause for the time the teacher trying to figure out who started the fight in the schoolyard.
> 
> ## Client Status Codes are Set by Your Domain's Registrar
> 
> | EPP Status Code | RDAP Status Mapping | What does it mean? | Should you do something? |
> | -- | -- | -- | -- |
> | clientHold | client hold | This status code tells your domain's registry to not activate your domain in the DNS and as a consequence, it will not resolve. It is an uncommon status that is usually enacted during legal disputes, non-payment, or when your domain is subject to deletion. | Often, this status indicates an issue with your domain that needs resolution. If so, you should contact your registrar to resolve the issue. If your domain does not have any issues, but you need it to resolve, you must first contact your registrar and request that they remove this status code. |
> 
> Source: https://mypdns.org/my-privacy-dns/matrix/-/issues/1835#note_3660

**Comment by testermania** [comment]
> > Kişilerin reddit gönderilerini gözden geçirirken (yani doğru kişiye bakıyorsam), bu tür şeyleri daha önce başka şeylerle de yapmışlar gibi görünüyor, bu yüzden göründüğü gibi onlardan fazla bir şey beklemem. hiçbir işe yaramıyorlar ve sadece popüler bir uygulamadan tekrar yararlanıyorlar.
> 
> Yaşları 5-7 arasında değişen birkaç hesapları var. Tüm Genel Orta Doğu adları.

They bought the domain name before you, why are you crying now? And you judge people by their names? You have to stop racism, your program will be forgotten after 1 month and will be on dusty shelves.

**Comment by krystian3w** [comment]
Bug your software never die - you learn childrens/grandchildren how edit your code?

**Comment by rcmaehl** [comment]
> > They have a few accounts ranging between 5-7 years old. All Generic Middle East names.
> 
> They bought the domain name before you, why are you crying now? And you judge people by their names? You have to stop racism, your program will be forgotten after 1 month and will be on dusty shelves.

Crying? No. Trying to prevent bait and switch malware/badware, yes. As proven in this thread, they were distributing badware after not being able to mirror the application.

Judge people by their names? No. They were all using what is basically "John Smith" or "John Doe". There's nothing wrong with pointing out a fact

**Comment by rcmaehl** [comment]
> And you judge people by their names?

Just for you I went and verified the person. They're running Mac 10.13.1 with Chrome 91 in India, Muthi

**Comment by iam-py-test** [comment]
> 
> 
> > And you judge people by their names?
> 
> Just for you I went and verified the person. They're running Mac 10.13.1 with Chrome 91 in India, Muthi using the ISP of National Internet Backbone. Got a bunch more metadata than that too, but I won't get more specific

How did you get that information?

**Comment by rcmaehl** [comment]
> > > And you judge people by their names?
> > 
> > 
> > Just for you I went and verified the person. They're running Mac 10.13.1 with Chrome 91 in India, Muthi using the ISP of National Internet Backbone. Got a bunch more metadata than that too, but I won't get more specific
> 
> How did you get that information?

Messaged one of their accounts and they clicked a link ¯\_(ツ)_/¯ even a basic web browser gives a ton of info in the header, add in javascript and other code and you can get a bunch more

**Comment by thepwrtank18** [comment]
how did this turn into a talk about racism

**Comment by micwoj92** [comment]
Open source is very political.

**Comment by rcmaehl** [comment]
> Open source is very political.

👀 Linux Kernel Email threads

**Comment by MDMCK10** [comment]
The site seems to actually be back this time, this entire thing is just a never ending battle at this point, and i find it slightly comical for some reason.
![image](https://user-images.githubusercontent.com/21245760/124485590-06513c80-ddad-11eb-8b79-26a8692886f8.png)
A quick look at the website reveals that the download button is completely gone for some reason, and the "Download" link on the navbar leads nowhere.
Also, was `© WhyNotWin11 is owned by Robert C. Maehl. We are not associated with WhyNotWin11 in any way. This is an unofficial free resource website.` always at the bottom of the page?

**Comment by iam-py-test** [comment]
I tried to access it & I got this error:
![image](https://user-images.githubusercontent.com/84232764/124486596-c1f97880-dd7b-11eb-84a8-76c739f2067a.png)

**Comment by spirillen** [comment]
God damned.... chrome can be true about itself!!!!

![image](https://user-images.githubusercontent.com/44526987/124487306-d7d46100-ddae-11eb-8e1a-8a461d74fc2c.png)

:smirk: :imp: :open_mouth:

**Comment by iam-py-test** [comment]
> 
> 
> God damned.... chrome can be true about itself!!!!
> 
> ![image](https://user-images.githubusercontent.com/44526987/124487306-d7d46100-ddae-11eb-8e1a-8a461d74fc2c.png)
> 
> 😏 👿 😮

What do you mean?
I tried using Firefox and got the same error.
Or are you talking about Chromium's not-so-secret anti-privacy

**Comment by spirillen** [comment]
> > ... chrome can be true about itself!!!!
> > ![image](https://user-images.githubusercontent.com/44526987/124487306-d7d46100-ddae-11eb-8e1a-8a461d74fc2c.png)
> > smirk imp open_mouth
> 
> What do you mean?
> I tried using Firefox and got the same error.
> Or are you talking about Chromium's not-so-secret anti-privacy tactics

google anti-privacy policy, where nothing is private to them, but all about them is private :smirk: 

You know...

`||goole.*^`

**Comment by iam-py-test** [comment]
> 
> 
> > > ... chrome can be true about itself!!!!
> > > ![image](https://user-images.githubusercontent.com/44526987/124487306-d7d46100-ddae-11eb-8e1a-8a461d74fc2c.png)
> > > smirk imp open_mouth
> > 
> > 
> > What do you mean?
> > I tried using Firefox and got the same error.
> > Or are you talking about Chromium's not-so-secret anti-privacy tactics
> 
> google anti-privacy policy, where nothing is private to them, but all about them is private 😏
> 
> You know...
> 
> `||goole.*^`

Ok. I think there is a typo in that filter though.

**Comment by micwoj92** [comment]
> Also, was `© WhyNotWin11 is owned by Robert C. Maehl. We are not associated with WhyNotWin11 in any way. This is an unofficial free resource website.` always at the bottom of the page?

Nope, it's recent addition.

**Comment by rcmaehl** [comment]
> > Also, was `© WhyNotWin11 is owned by Robert C. Maehl. We are not associated with WhyNotWin11 in any way. This is an unofficial free resource website.` always at the bottom of the page?
> 
> Nope, it's recent addition.

LGTM.

**Comment by thepwrtank18** [comment]
their download doesn't even work anymore, just goes back to the home page

**Comment by rcmaehl** [comment]
I'm going to go ahead and close this out. While some people may be confused still, I am happy with the disclaimer. I wish they would have replied via the forwarding service and other contact methods, but I'll continue to keep an eye on the situation.

**Comment by thepwrtank18** [comment]
We got a good ending (hopefully it stays that way). Good the situation got resolved.

**Comment by spirillen** [comment]
I thinks we all (BlackList maintainers) should remember to [REMOVE](https://mypdns.org/my-privacy-dns/matrix/-/issues/1835#note_3821) whynotwin11.com from our blacklists.... right?

**Comment by iam-py-test** [comment]
Should we report FPs to the AV detections on VirusTotal?

**Comment by spirillen** [comment]
> Should we report FPs to the AV detections on VirusTotal? (there is now 4)
> I have reported to FortiGuard

VT is only operating a collection of list / results from external sources, such as @mitchellkrogza's https://github.com/mitchellkrogza/Phishing.Database. 

Requesting removal from these lists become the site/domain owners own headache.

**Comment by iam-py-test** [comment]
> > Should we report FPs to the AV detections on VirusTotal? (there is now 4)
> > I have reported to FortiGuard
> 
> VT is only operating a collection of list / results from external sources, such as @mitchellkrogza's https://github.com/mitchellkrogza/Phishing.Database.
> 
> Requesting removal from these lists become the site/domain owners own headache.

If they are legit, that should not be hard & they might want to

**Comment by AlexanderEDavis** [comment]
![image](https://user-images.githubusercontent.com/9097891/130323043-dd0b212b-2e83-4d49-bc3a-98bcdcb1ffeb.png)

They admit the program isn't theirs, they might be expecting you to buy the domain from them for a 'reasonable sum'...

**Comment by HackerDaGreat57** [comment]
The ads and the articles at the bottom of the page make it look like the domain was obtained for free.
![image](https://user-images.githubusercontent.com/70416002/131394508-c5f66fbf-c3ff-47fc-9439-a1bd5ce51fe8.png)

**Comment by krystian3w** [comment]
More likely, someone is trying to recoup the cash for paying for the hosting/domain.

**Comment by 0xpr03** [comment]
> The ads and the articles at the bottom of the page make it look like the domain was obtained for free.

You mean the 3€/month webhosting + 10€ domain/year ?

**Comment by krystian3w** [comment]
Yeah ...in Poland ~60-300 zł (13,50-66,50 €) at year if we  found good chip hosting/domain renew.

**Comment by JesseChisholm** [comment]
re: waiting for you to buy the domain (for a _reasonable_ fee)
On some registries, you can at least put in for a backorder, if they ever forget to renew.
